bitcoin_executor/sources/interpreter.move

```diff
@@ -1,21 +1,48 @@
 module bitcoin_executor::interpreter;
-use bitcoin_executor::stack::{Stack, Self};
-use bitcoin_executor::reader::{ScriptReader, Self};
 
+use bitcoin_executor::reader::{Self, ScriptReader};
+use bitcoin_executor::stack::{Self, Stack};
+use std::unit_test::assert_eq;
 
 // Opcodes
-const OP_DUP: u8 = 76;
+// push value
+const OP_0: u8 = 0x00;
+const OP_FALSE: u8 = OP_0;
+const OP_PUSHBYTES_1: u8 = 0x01;
+// TODO: do we need to add all the in-between constants?
+const OP_PUSHBYTES_75: u8 = 0x4b;
+const OP_PUSHDATA1: u8 = 0x4c;
+const OP_PUSHDATA2: u8 = 0x4d;
+const OP_PUSHDATA4: u8 = 0x4e;
+const OP_1: u8 = 0x51;
+const OP_TRUE: u8 = OP_1;
+const OP_2: u8 = 0x52;
+const OP_3: u8 = 0x53;
+const OP_4: u8 = 0x54;
+const OP_5: u8 = 0x55;
+const OP_6: u8 = 0x56;
+const OP_7: u8 = 0x57;
+const OP_8: u8 = 0x58;
+const OP_9: u8 = 0x59;
+const OP_10: u8 = 0x5a;
+const OP_11: u8 = 0x5b;
+const OP_12: u8 = 0x5c;
+const OP_13: u8 = 0x5d;
+const OP_14: u8 = 0x5e;
+const OP_15: u8 = 0x5f;
+const OP_16: u8 = 0x60;
+// stack ops
+const OP_DUP: u8 = 0x76;
```

Comment:
I've added the op codes in a single list, so let's update this PR after merging https://github.com/gonative-cc/sui-native/pull/46

Reply:
ok